interface/src/app/volcano-plot/view.js

```diff
@@ -0,0 +1,34 @@
+angular.module('adage.volcano-plot.view', [
```

Comment:
I'm just curious - why the hyphen instead of the camelCase for ``volcano-plot``?